src/libraries/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -5,7 +5,6 @@
     <TargetFrameworks>netstandard2.0-Windows_NT;netstandard2.0;net461-Windows_NT;$(NetFrameworkCurrent)-Windows_NT</TargetFrameworks>
     <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants>$(DefineConstants);WINHTTPHANDLER_DLL</DefineConstants>
```

Comment:
This constant was only used in `NoWriteNoSeekStreamContent` which was recently moved to WinHttpHandler dll and it lost it's meaning.
I removed it within this PR, but I can split it into it's own PR if preferred.

Reply:
Reverted WinHttpHandler changes.